src/tests/integration/TrackCard.ct.tsx

```diff
@@ -30,25 +39,23 @@ function getTrackData(overrides: Partial<Track> = {}): Track {
 
 const mockSetCurrentPlay: SetCurrentPlay = (audio) => {}
 
-function getByTestIdTemple(component: MountResult, testId: string) {
-  return component.locator(`[data-testid="${testId}"]`)
-}
-
 function getTestTrackCard(overrides: Partial<Track> = {}) {
   const data = getTrackData(overrides)
   return (
-    <CardIdentifierProvider>
-      <ModalProvider>
-        <TrackCard
-          data={data}
-          setCurrentPlay={mockSetCurrentPlay}
-          currentPlay={null}
-        />
-      </ModalProvider>
-    </CardIdentifierProvider>
+    <Provider store={store}>
+      <TrackCard
+        data={data}
+        setCurrentPlay={mockSetCurrentPlay}
+        currentPlay={null}
+      />
+    </Provider>
   )
 }
 
+function getByTestIdTemple(component: MountResult, testId: string) {
+  return component.locator(`[data-testid="${testId}"]`)
```

Comment:
```suggestion
function getByTestIdTemple(component: MountResult, testId: TestId) {
  return component.locator(`[data-testid="${testId}"]`)
```
Чи немає в бібліотеці яку ти використовуєш метода `getByTestId`?
Також можна обмежити щоби `testId` була не просто строка, test id із списку валідних